### PR TITLE
fix(BaseTrendWidget): set full data link

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
스크린샷 2023-01-06 오후 4 49 44

### CODE REVIEW

File: src/services/dashboards/widgets/budget-status/BudgetStatus.vue [hunk 1]
```diff
@@ -206,6 +206,8 @@ defineExpose<WidgetExpose<Data>>({
         }
         .legend-wrapper {
             @apply text-gray-600;
+            position: absolute;
```
Comment (minseolee) [inline src/services/dashboards/widgets/budget-status/BudgetStatus.vue:209]: It is hard to find `position: relative` on parent element.
Is it ok ?

**Review by minseolee — APPROVED** [review]
Left a comment ~ 🤔 
Else things are look good to me

**Review by sulmoJ — APPROVED** [review]
LGTM!!

File: src/services/dashboards/widgets/budget-status/BudgetStatus.vue (hunk 1 above)
Comment (yuda110) [reply src/services/dashboards/widgets/budget-status/BudgetStatus.vue:209]: I missed it! Thanks!

**Review by song2kim — APPROVED** [review]
LGTM~!

**Pull request merged**